tests/tools.py

```diff
@@ -132,3 +132,14 @@ def verify_all_nodes(self, test_case):
       tested_nodes += 1
 
     return tested_nodes
+
+
+def Constant(name):
```

Comment:
What do you think of changing `MarkChecker.view_node` to replace `node.__class__.__name__` with `get_node_name(node)` which is something like this:
```
def get_node_name(node):
   name = node.__class__.__name__
   return "Constant" if name in ("Num", "Str", "NameConstant") else name
```
Then we can change code throughout to expect things like `Constant:17` or `Constant:'hello'`, which seems a bit simpler.